documentation/bin/createDocu.groovy

```diff
@@ -462,6 +584,13 @@ if( ! steps) {
     System.err << "[INFO] Generating docu only for step ${steps.size > 1 ? 's' : ''} ${steps}.\n"
 }
 
+// find all the stages that we have to document
+Map stages = Helper.resolveDocuRelevantStages(gse, stepsDir)
+
+// retrieve default conditions for steps
+//ToDo: allow passing config file name via parameter
+Map stageConfig = Helper.getYamlResource('piper-stage-config.yml')
```

Comment:
This file is not present inside the `resources` folder.

Reply:
yes, depends on the lib. That is why I wanted to wait for the additional parameter flexibility of the generator. This has to go into a parameter which activates/de-activates this capability.